pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -162,7 +165,8 @@ func (n *NetworkPolicyController) toAntreaPeerForCRD(peers []v1alpha1.NetworkPol
 	}
 	var ipBlocks []controlplane.IPBlock
 	var fqdns []string
-	var clusterSetScopeSelectors []*antreatypes.GroupSelector
+	var labelIdentities []uint32
+	uniqueLabelIDs := map[uint32]struct{}{}
```

Comment:
Does `sets.Int` work better?

Reply:
`sets.Int` and `sets.Int32` does not take uint32 values. Under the hood I believe this is the same implementation as the sets package

Reply:
Ok. It is just more convenient.